googletest/include/gtest/gtest.h

```diff
@@ -658,6 +661,13 @@ class GTEST_API_ TestResult {
   // Increments the death test count, returning the new count.
   int increment_death_test_count() { return ++death_test_count_; }
 
+  // Increments the successful assert count, returning the new count.
+  size_t increment_success_assert_count(size_t v = 1) { return success_assert_count_ += v; }
```

Comment:
Should just return void, I dont see where the return is actually used

Reply:
Personally I think it makes a better API being able to read the old value while incrementing,
even though it is not used, and it is a common way to implement such functions. 
But I can remove it.